devel/README.md

```diff
@@ -19,7 +19,7 @@ To get up and running you'll need
 Chainloop requires some configuration to be in place such as
 
 - PostgreSQL 14 connection from the Control plane
-- Access to a secrets manager. Currently Hashicorp Vault and AWS secret manager are supported.
+- Access to a secrets manager. Currently Hashicorp Vault, AWS secret manager, GCP secret manager and Azure KeyVault are supported.
```

Comment:
Currently Hashicorp Vault, AWS Secrets Manager, Google Cloud Secret Manager, and Azure KeyVault.